src/actions/events.js

```diff
@@ -1,15 +1,74 @@
+import database from "../firebase/firebase";
+
+export const setEvents = events => ({
+  type: "SET_EVENTS",
+  events,
+});
+
+export const startSetEvents = () => (dispatch, getState) => {
+  const { uid } = getState().auth;
+
+  return database
+    .ref(`users/${uid}/events`)
+    .once("value")
+    .then(snapshot => {
+      const events = [];
+
+      snapshot.forEach(childSnapshot => {
+        events.push({
```

Comment:
Mutating is usually a bad idea. Please, use `map` instead of `forEach`. `forEach` is good if you don't want to get a new array but you want to make some actions or run some functions for each element in your array.

Reply:
I cannot change it, it's a firebase method.